mysql-test/suite/innodb/r/partition.result

```diff
@@ -437,6 +437,7 @@ DROP TABLE t1;
 #
 # Bug#21957001 INNODB: FAILING ASSERTION: 0 IN FILE HA_INNOPART.CC LINE 3526
 #
+call mtr.add_suppression("\\[Warning\\] .*MY-\\d+.* Cannot save statistics for table `test`\\.`t1`");
```

Comment:
Is this suppression missing for upstream too?

Reply:
No, `Cannot save statistics` is Percona warning